pipeline_schedules.go

```diff
@@ -35,23 +35,26 @@ type PipelineSchedulesService struct {
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/pipeline_schedules.html
 type PipelineSchedule struct {
-	ID           int        `json:"id"`
-	Description  string     `json:"description"`
-	Ref          string     `json:"ref"`
-	Cron         string     `json:"cron"`
-	CronTimezone string     `json:"cron_timezone"`
-	NextRunAt    *time.Time `json:"next_run_at"`
-	Active       bool       `json:"active"`
-	CreatedAt    *time.Time `json:"created_at"`
-	UpdatedAt    *time.Time `json:"updated_at"`
-	Owner        *User      `json:"owner"`
-	LastPipeline struct {
-		ID     int    `json:"id"`
-		SHA    string `json:"sha"`
-		Ref    string `json:"ref"`
-		Status string `json:"status"`
-	} `json:"last_pipeline"`
-	Variables []*PipelineVariable `json:"variables"`
+	ID           int                           `json:"id"`
+	Description  string                        `json:"description"`
+	Ref          string                        `json:"ref"`
+	Cron         string                        `json:"cron"`
+	CronTimezone string                        `json:"cron_timezone"`
+	NextRunAt    *time.Time                    `json:"next_run_at"`
+	Active       bool                          `json:"active"`
+	CreatedAt    *time.Time                    `json:"created_at"`
+	UpdatedAt    *time.Time                    `json:"updated_at"`
+	Owner        *User                         `json:"owner"`
+	LastPipeline *PipelineScheduleLastPipeline `json:"last_pipeline"`
+	Variables    []*PipelineVariable           `json:"variables"`
+}
+
+// PipelineScheduleLastPipeline represents the last pipeline ran by schedule
+type PipelineScheduleLastPipeline struct {
```

Comment:
```suggestion
type LastPipeline struct {
```
I guess that should be long enough 😏

Reply:
Sounds good. I am in the habit of usually prepending the parent name to have uniqueness. :)

Reply:
Yeah fully understand, but when not needed I prefer to keep things a little shorter 😊